starboard/build/config/BUILDCONFIG.gn

```diff
@@ -90,6 +90,10 @@ import("//starboard/build/platform_path.gni")
 # TODO(b/272790873): Set host_cpu to x86 for 32 bit builds.
 if (target_cpu == "x86" || target_cpu == "arm") {
   _host_toolchain_cpu = "x86"
+} else
+# TODO(b/292107465): Set host_cpu to x64 for arm64 builds.
+if (target_cpu == "arm64") {
```

Comment:
Can you put this in a higher level else if instead of nesting it?

Reply:
Discussed offline, moving the comment inside the if block, to make `} else if ()` in one line.